node_modules/oae-admin/importusers/js/importusers.js

```diff
@@ -166,9 +166,8 @@ define(['jquery', 'oae.core', 'jquery.fileupload', 'jquery.iframe-transport'], f
                     'backdrop': 'static'
                 });
 
-                // Hide the spinner icon using jQuery
-                // @see https://github.com/FortAwesome/Font-Awesome/issues/729
```

Comment:
Not sure I understand this comment. The reference is to a bug that has been fixed, and it describes a work-around that we're no longer using. It's completely irrelevant for the new code.